app/controllers/openstax/api/v1/api_controller.rb

```diff
@@ -46,8 +48,12 @@ def session_user?
           !!current_session_user && doorkeeper_token.blank?
         end
 
+        def set_date_header
+          response.date = Time.now unless response.date?
```

Comment:
this works?  is this the cache control header?  ok to use it for this purpose (other than making a different custom header field?)